src/rules/at-rule-no-unknown/README.md

```diff
@@ -0,0 +1,86 @@
+# at-rule-no-unknown
+
+Disallow unknown at-rules.
+
+```css
+    @media (max-width: 960px) {}
+/** ↑
+ * At-rules like this */
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+@unknown {}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+@charset "UTF-8";
+```
+
+```css
+@CHARSET "UTF-8";
+```
+
+```css
+@media (max-width: 960px) {}
+```
+
+```css
+@font-feature-values Font One { 
+  @styleset {} 
+}
+```
+
+## Optional options
+
+### `ignoreAtRules: ["array", "of", "at-rules"]`
+
+Allow unknown at-rule.
+
+Given:
+
+```js
+["unknown"]
+ 
+```
+
+The following patterns are considered warnings:
+
+```css
+@unknown-at-rule {}
+```
+
+```css
```

Comment:
Lets drop this example as it's not immediately obvious what's going on as contains both a valid and invalid pattern.